modules/setting/database.go

```diff
@@ -42,12 +42,14 @@ var (
 		DBConnectRetries  int
 		DBConnectBackoff  time.Duration
 		MaxIdleConns      int
+		MaxOpenConns      int
 		ConnMaxLifetime   time.Duration
 		IterateBufferSize int
 	}{
 		Timeout:         500,
-		MaxIdleConns:    0,
-		ConnMaxLifetime: 3 * time.Second,
+		MaxIdleConns:    2,
```

Comment:
`MaxIdleConns` doesn't need a default value here.

Reply:
Why not?

Reply:
Because if I'm not mistaken it's overwritten a few line below, either on if {} or else {}. I'll check again.

Reply:
Yes, there is this a few lines below:
```
if Database.UseMySQL {
	Database.ConnMaxLifetime = sec.Key("CONN_MAX_LIFE_TIME").MustDuration(3 * time.Second)
		Database.MaxIdleConns = sec.Key("MAX_IDLE_CONNS").MustInt(0)
		Database.ConnMaxLifetime = sec.Key("CONN_MAX_LIFE_TIME").MustDuration(3 * time.Second)
	} else {
		Database.MaxIdleConns = sec.Key("MAX_IDLE_CONNS").MustInt(2)
		Database.ConnMaxLifetime = sec.Key("CONN_MAX_LIFE_TIME").MustDuration(0)
	}
```
A default here will not have any effect, I believe.

Reply:
```suggestion
```